test/cypress/integration/lazy-chat.test.js

```diff
@@ -0,0 +1,46 @@
+import { crispWebsiteId } from '../../../src/client/static/data/app.json'
+
+const lazyChatButtonIdentity = '[data-test="lazyChatButtonIdentity"]'
+const lazyChatOptInIdentity = '[data-test="lazyChatOptInIdentity"]'
+
+describe('Lazy chat', () => {
+  beforeEach(() => {
+    cy.visit('/')
+  })
+
+  it('is only present if Crisp ID is set', () => {
+    if (!!crispWebsiteId) {
+      cy.get(lazyChatButtonIdentity).should('be.visible')
+    } else {
+      cy.get(lazyChatButtonIdentity).should('not.exist')
+    }
+  })
+
```

Comment:
I think you need to make all of the tests after this point conditional. They only apply if the chat is enabled, so:
```js
if (!chatIsEnabled) {
  return
}
```